rustls/src/msgs/enums.rs

```diff
@@ -139,6 +139,11 @@ enum_builder! {
     /// The `NamedCurve` TLS protocol enum.  Values in this enum are taken
     /// from the various RFCs covering TLS, and are listed by IANA.
     /// The `Unknown` item is used when processing unrecognised ordinals.
+    ///
+    /// This enum is used for recognizing elliptic curve parameters advertised
+    /// by a peer during a TLS handshake. It is **not** a list of curves that
+    /// Rustls supports. Rustls only supports ECDSA NISTP256, ECDSA NISTP384
+    /// and Ed25519.
```

Comment:
Maybe link to https://docs.rs/rustls/latest/rustls/kx_group/index.html (however that is achieved, is it `[crate::kx_group]`?) to avoid this getting out of date?

Reply:
Good idea, thanks. Fixed up.